.gitpod.yml

```diff
@@ -22,9 +22,9 @@ github:
 
 # List the start up tasks. You can start them in parallel in multiple terminals. See https://www.gitpod.io/docs/config-start-tasks/
 tasks:
-  - before: nvm use default
+  - before: echo "nvm use default" >> ~/.bashrc
     init: yarn && alias near=./node_modules/near-shell/bin/near
-    command: gp open README-Gitpod.md && yarn dev
+    command: source ~/.bashrc gp open README-Gitpod.md && yarn dev
```

Comment:
Does this work when you test it out?  It seems like you need a `;` between `source ~/.bashrc` and the rest.

Reply:
it does work when I tested it out... let me try adding a semi colon to see if that causes any issues

Reply:
everything still works! I'll go ahead and add the semi-colon if that is best practice